VideoLocker/res/values/strings.xml

```diff
@@ -360,10 +361,10 @@
 
     <!-- Title placeholder in adding a new post -->
     <string name="discussion_post_title">Title</string>
-    <!-- Add post button title in adding a new discussion -->
-    <string name="discussion_add_post_topic_label">Topic</string>
-    <!-- Add post button title in adding a new discussion -->
+    <!-- Topic selection placeholder -->
     <string name="discussion_add_post_choose_a_topic">Choose a topic…</string>
+    <!-- Label for selected topic when creating a post -->
+    <string name="discussion_add_post_topic_selection">Topic: %s</string>
```

Comment:
Instead of ``%s`` it would be better to use named arguments, for example ``Topic: {topic}`` since they're better for translators. See ``ResourceUtil`` for the associated helper